File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    s = s.split(' ')
    
    for i in range(len(s)):
        for j in range(len(s[i])):
            if (j+1) % 2 == 0:
                answer += s[i][j].lower()
            else:
                answer += s[i][j].upper()
        if i == len(s)-1:
            break
        else:
            answer+= ' '
            
    return answer

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String[] words = s.split(" ", -1);
        String answer = "";
        
        for(String word : words){
            for(int i = 0; i < word.length(); i++){
                if(i % 2 == 0){
                    answer += Character.toUpperCase(word.charAt(i));
                }else{
                    answer += Character.toLowerCase(word.charAt(i));
                }
            }
            answer += " ";
        }
        
        answer = answer.substring(0, answer.length()-1);
        
        return answer;
    }
}